.eslintrc

```diff
@@ -106,6 +107,14 @@
         "import/no-named-as-default": "off"
       }
     },
+    {
+      "files": [
+        "./packages/front-end/**/*.ts*"
+      ],
+      "rules": {
+        "no-restricted-imports": ["error", {"patterns": ["..*"]}]
```

Comment:
I think this might be too restrictive but not sure what the team thinks. Would be great to bring up at the eng. roundtable

Reply:
Will do!